packages/components/button-group/src/button-group.ts

```diff
@@ -0,0 +1,58 @@
+import { type ButtonFill, type ButtonSize } from '@sl-design-system/button';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import styles from './button-group.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-button-group': ButtonGroup;
+  }
+}
+
+type ButtonLike = HTMLElement & {
+  disabled: boolean;
+  fill?: ButtonFill;
```

Comment:
Will it be applicable to a toggle button as well?

Reply:
Yes, toggle-button has that property/value as well. If if somebody slots something that doesn't have that, it's basically just a NOOP.